src/components/QueryBuilder/hooks/useTranslatedConfig.tsx

```diff
@@ -16,9 +16,24 @@ import { TimePicker } from '../../inputs/TimePicker';
 import { DatePicker } from '../../DatePicker';
 import { DateTimePicker } from '../../DateTimePicker';
 
+const transformListValuesToOptions = (listValues: any) => {
+    return Array.isArray(listValues)
+        ? listValues.map(listValue => ({
+            value: listValue.value,
+            label: listValue.title,
+        }))
+        : Object.entries(listValues || {}).map(([value, title]) => ({
+            value,
+            label: title,
+        }));
+};
+
 export const useTranslatedConfig = (): Config => {
     const { formatMessage } = useSafeIntl();
     const theme = useTheme();
+    const handleChangeMultiselect = useCallback((newValue: any, setValue: (value: any) => void) => {
```

Comment:
From the code I gather the correct typing looks like:
```suggestion
    const handleChangeMultiselect = useCallback((newValue: string | string[], setValue: (value:string[]) => void) => {
```